Create Product Announcements.md (#2061)
PostHog/posthog.com: contents/handbook/growth/marketing/product-announcements.md

---
title: Product announcements
sidebar: Handbook
showTitle: true
---

> Marketing takes responsibility for coordinating and publicising PostHog news, including product announcements. 

Reviewer: ```suggestion
Marketing takes responsibility for coordinating and publicizing news about PostHog, including product announcements. 
```

"PostHog News" seems confusing as it's an internal concept we use to refer to the updates on Monday.

However, not all announcements require the same level of marketing support. We therefore group them into tiers which help us decide what level of investment is required from the Marketing team. Which tier applies is agreed upon by the Marketing and Product teams and agreement should be sought as early as possible if Marketing support is expected.

### Tier 3: Service features
Service features are those which introduce changes which we do not expect to have a noticeable impact for a majority of users.

An example of a service feature would include bug fixes or minor performance improvements. 

Marketing support for a service feature would typically include: 

- [x] - Including the news in the PostHog newsletter.

> - [x] - Including the news in PostHog Array updates.

Reviewer: PostHog Array is published by Product/Engineering anyways, so might be better to delegate to the release guide there?

### Tier 2: Core features
Core features are those which introduce changes which we expect to have a noticeable impact for a larger subset of users, but not a majority. 

An example of a core feature would include a new plugin which integrates PostHog with a third-party platform, or a cosmetic refresh of an existing feature. 

Marketing support for a core feature would typically include:

> - [x] - Creating a blog announcement about the feature within one week of launch.

Reviewer: Not sure if at this stage we'd want to create a blog post for UI refreshes or new plugins, perhaps clarify it's more of a judgement call on a case-by-case basis

Author: Added a line about this in the preamble.

> - [x] - Sharing the blog announcement with the Slack community via the #Editorial channel.

Reviewer: ```suggestion
- [x] - Sharing the blog announcement with the Slack community via the #editorial channel.
```

- [x] - Sharing the blog announcement on PostHog's social media channels.
- [x] - Sharing links to the social media announcement internally via Slack, so colleagues can amplify them.
- [x] - If relevant, sharing the blog announcement with other external parties, such as integration partners or PPC agencies.
- [x] - Updating any required assets on PostHog.com. 
- [x] - Including the news in the PostHog newsletter.
- [x] - Including the news in PostHog Array updates.

### Tier 1: Major features
Major features are those which introduce changes which we expect to have noticeable impact for a majority of users or which would represent reasons why prospective users would choose PostHog.

An example of a major feature would include the creation of a new tool, such as session recordings, or a fundamental overhaul of an existing feature. 

Marketing support for a major feature would typically include:

- [x] - Creating a blog announcement about the feature within 24 hours of launch.

> - [x] - Sharing the blog announcement with the Slack community via the #General channel.

Reviewer: ```suggestion
- [x] - Sharing the blog announcement with the Slack community via the #general channel.
```
